packages/repository-json-schema/src/build-schema.ts

```diff
@@ -230,6 +230,10 @@ export function stringTypeToWrapper(type: string | Function): Function {
       wrapper = Buffer;
       break;
     }
+    case 'binary': {
```

Comment:
Why cant we use buffer from last case ? Do we really need binary type ?

Reply:
We need to handle the binary type due to an error with blob data. Please have a look at [this](https://github.com/loopbackio/loopback-next/pull/9077).